cypress/e2e/awx/resources/inventoryHost/inventoryHostSmart.cy.ts

```diff
@@ -60,9 +58,22 @@ describe('Inventory Host Tab Tests for smart inventory', () => {
     //5) Cancel the job and assert that it has been canceled
   });
 
-  it.skip('confirm that edit host button is missing from the host tab list of an inventory', () => {});
+  it('confirm that edit host button is missing from the host tab list of an inventory', () => {
+    //navigate to smart inventory host list
+    //verify edit button is missing
+    testMissingButton('inventory_host', inventory, 'list', 'edit-host');
+  });
 
-  it.skip('confirm that delete host button is missing from the host tab list of an inventory', () => {});
+  it('confirm that delete host button is missing from the host tab list of an inventory', () => {
+    //navigate to smart inventory host list
+    //verify action dropdown contain only delete host button is missing
+    const missingValue = `[data-cy="actions-column-cell"] [data-cy="actions-dropdown"]`;
```

Comment:
You are using contains wrong, you are doing:

contains('tab', 'get selector')

While it should be:
Contains('selector', 'string value')

Because get selector will be treated like string value that the cypress will try to search, it will not find it. But it will also not find it in inventories, where those buttons and tabs actualy are. So the test always pass, even when it should not.

I suggest you should add also test for buttons and tabs for inventories where there really should be button and tab. Then, you can use the same selectors but with the function should('not.exist') for inventories where those tabs and buttons should missing.